keps/1-kubernetes-enhancement-proposal-process.md

```diff
@@ -141,11 +130,16 @@ proposing governance changes. However, as changes start impacting other SIGs or
 the larger developer community outside of a SIG, the KEP process should be used
 to coordinate and communicate.
 
-### KEP Template
+Enhancements that have major impacts on multiple SIGs should use the KEP process.
+A single SIG will own the KEP but it is expected that the set of approvers will span the impacted SIGs.
+The KEP process is the way that SIGs can negotiate and communicate changes that cross boundaries.
+
+KEPs will also be used to drive large changes that will cut across all parts of the project.
+These KEPs will be owned by SIG-architecture and should be seen as a way to communicate the most fundamental aspects of what Kubernetes is.
```

Comment:
I don't believe it's needed to call this out explicitly. A cross cutting KEP will probably be managed by SIG Architecture but, like the cloud provider extraction, another SIG like API Machinery might manage the KEP itself

Reply:
I don't think the cloud provider KEP cuts across *all* parts of the project.  It only involves a limited number of SIGs.